### PR TITLE
fix: doc links

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
### Describe your changes:

I worked on fixing the doc links.

#
### Type of change:

- [x] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/constants/docs.constants.ts [hunk 1]
```diff
@@ -12,62 +12,64 @@
...
 export const LOCAL_DEPLOYMENT =
-  'https://docs.open-metadata.org/quick-start/local-deployment';
+  'https://docs.open-metadata.org/v1.0.0/quick-start/local-deployment';
 
 export const OMD_SLACK_LINK =
   'https://join.slack.com/t/openmetadata/shared_invite/zt-1r1kv175f-9qM5eTB39MF6U2DBhZhWow';
 
 export const DATA_INSIGHTS_PIPELINE_DOCS =
-  'https://docs.open-metadata.org/openmetadata/data-insight#data-insights';
+  'https://docs.open-metadata.org/v1.0.0/features/data-insight#data-insights';
```
[the remark this answers is not in the document]
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/constants/docs.constants.ts:63]: @chirag-madlani, the path is different here that's why we have to update it.

[the remark this answers is not in the document]
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/constants/docs.constants.ts:63]: Yes we can update the path but don't want to mention version is my point

[the remark this answers is not in the document]
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/constants/docs.constants.ts:63]: Yes, I will remove the version as the doc is handling it.

[the remark this answers is not in the document]
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/constants/docs.constants.ts]: Done.

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**